web/src/cypress/support/config.ts

```diff
@@ -86,6 +89,9 @@ function merge(dst: Config, src: ConfigInput): Config {
   return dst
 }
 
+/*
+ * Updates the current config with newCfg
```

Comment:
Can we format these a little differently?

```
// updateConfig updates the current config with newConfig
```

That would match the Go style, and the majority of our js/ts function descriptions.
For this one it might also be helpful to indicate it merges it with the current though so it's obvious what "update" means (without looking at "set")

```
// updateConfig updates the current config by merging the provided newCfg values
```

Something like that

Reply:
That sounds good. I'm used to the convention I learned in college where `// comments` are for single line items, and
```
/*
 * block comments
 */
```
are for function definitions.

Reply:
That definitely makes sense. For instance, above the test def [here](https://github.com/target/goalert/pull/608/files#diff-199d37c4de5ee0e358ded986a41688bbR11) I'm using `// inline` comments